src/resources/2020/Guzzlr.ts

```diff
@@ -198,13 +198,13 @@ export const platinumCocktailToIngredient = invertMap(
   ingredientToPlatinumCocktail
 );
 
-export function getCheapestPlatinumCocktail(freeCraft: boolean = true): Item {
+export function getCheapestPlatinumCocktail(freeCraft = true): Item {
   const defaultCocktail = [$item`Dish of Clarified Butter`, $item`Buttery Boy`];
   if (freeCraft) {
     return (maxBy(
       Array.from(ingredientToPlatinumCocktail),
       (ingredientAndCocktail: [Item, Item]) =>
-        mallPrice(ingredientAndCocktail[0])
+        Math.min(...ingredientAndCocktail.map(mallPrice))
```

Comment:
I would test this--ASH functions don't know how to handle being given extra inputs, and so it'll try to interpret it as mallPrice(element, index, array) and have a bad time

Reply:
```suggestion
        Math.min(...ingredientAndCocktail.map((item) => mallPrice(item)))
```